pkg/api/pullrequestpayloadqualification/v1/types.go

```diff
@@ -0,0 +1,93 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/ci-tools/pkg/api"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// PullRequestPayloadQualificationRun represents the intent to run a battery of OCP release
+// payload validating jobs
+type PullRequestPayloadQualificationRun struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata"`
+
+	Spec   PullRequestPayloadTestSpec   `json:"spec"`
+	Status PullRequestPayloadTestStatus `json:"status"`
+}
+
+// PullRequestPayloadTestSpec specifies for which PR the payload qualification run was requested
+// and the list of individual jobs that should be executed
+type PullRequestPayloadTestSpec struct {
+	PullRequest PullRequestUnderTest        `json:"pullRequest"`
+	Jobs        []PullRequestPayloadJobSpec `json:"jobs"`
+}
+
+// PullRequestUnderTest describes the state of the repo that will be under test
+// This is a combination of the PR revision and base ref revision. Tested code
+// is the specific revision of the PR merged into the base branch with
+// a specific branch as a HEAD
+type PullRequestUnderTest struct {
+	// Org is something like kubernetes or k8s.io
+	Org string `json:"org"`
+	// Repo is something like test-infra
+	Repo string `json:"repo"`
+	// BaseRef identifies the target branch for the PR
+	BaseRef string `json:"baseRef"`
+	// BaseSHA identifies the HEAD of BaseRef at the time
+	BaseSHA string `json:"baseSHA"`
+
+	PullRequest PullRequest `json:"pr"`
+}
+
+// PullRequest identifies a pull request in a repository
+type PullRequest struct {
+	Number int    `json:"number"`
+	Author string `json:"author"`
+	SHA    string `json:"sha"`
+	Title  string `json:"title"`
+}
+
+// PullRequestPayloadJobSpec specifies an individual job that should be executed
+type PullRequestPayloadJobSpec struct {
+	ReleaseJobSpec `json:"releaseJobSpec"`
+}
+
+// ReleaseJobSpec identifies the release payload one qualification test to execute. In this context,
+// "test" means one item in the specified ci-operator configuration file.
+type ReleaseJobSpec struct {
+	// CiOperatorConfig identifies the ci-operator configuration with the test
+	CiOperatorConfig api.Metadata `json:"ciOperatorConfig"`
+	// Test is the name of the test in the ci-operator configuration
```

Comment:
is this a Job?

Reply:
Not alone. Whole `ReleaseJobSpec` instance corresponds to one Prowjob configuration.